### PR TITLE
WIP [W3B-122][W3B-123] : Rewrite Reward Mechanism

### PR DESCRIPTION
**Pull request by tmavroeid**

### CODE REVIEW

File: docs/project/reward-mechanism.mdx [hunk 1]
```diff
@@ -5,13 +5,18 @@ sidebar_label: Reward Mechanism
...
 ---
 
-Rewarding the members of our network for sharing their local weather data is a crucial part of our system (**currently on testnet**).
+## Introduction
+
+:::important
+The Reward Mechanism is currently at **version 1.0**. This page serves as both an informational resource about its current status and a glimpse into its future.
+:::
+
+- Also include a short overview of the timeline here
```
Comment (NikosTsiligaridis) [inline docs/project/reward-mechanism.mdx:14]: `also include a short overview of the timeline here`
This is a placeholder :stuck_out_tongue: 
Please include an intro outlining:
- What is the reward mechanism
- What purpose does it serve
- Something along the lines of: 
At present the Reward Mechanism stands at version 1.0. This version of the RM allocates rewards by evaluating only basic station data adequacy (whether a weather station sent enough data for the period or not). 
In RM v1.5 the QoD v1.0 and PoL v1.0 mechanisms are integrated into the pipeline of rewards. With QoD the weather data quality is evaluated using more advanced checks, while PoL verifies the location data provided by the weather station.
In RM v2.0 ...
[pls expand the above as you see fit]

Following is an in-depth view of the inner workings of the current RM version, as well as future milestones.

Comment (tmavroeid) [reply docs/project/reward-mechanism.mdx:14]: I understood the placeholder part, no worries.

File: docs/project/quality-of-data.mdx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+
+QoD is an algorithm evaluates the quality of weather data provided by a weather station. 
+The score calculated indicates the confidence level about the quality of the weather data received by a station. 
+We need meaningful and usable data and the QoD score is an attempt to quantify this metric. 
+The end goal is to encourage weather station owners to do their best to comply with the Network's guidelines in order to consistently 
+achieve the best QoD score possible.
+
+:::note
+QoD is a rational number in the range [0,1]
```
[the remark this answers is not in the document]
Comment (NikosTsiligaridis) [reply docs/project/quality-of-data.mdx]: @ipanagiv still pending

File: docs/project/cell-capacity.mdx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+
+**Cell Capacity (CC)** is a parameter that is used to define the maximum number of stations that may receive rewards in a specific cell.
+
+Each cell has a predefined cell capacity that depends on its geospatial characteristics. 
+The cell's capacity is the maximum number of stations that may receive rewards in a specific cell. 
+If 'N' weather stations are rewardable in a cell with a capacity CC during a specific day, if N>CC, then only the first N stations (ordered first by their rewards score defined below, and then by seniority) will receive rewards.
+
+Related Articles:
```
[the remark this answers is not in the document]
Comment (NikosTsiligaridis) [reply docs/project/cell-capacity.mdx]: @ipanagiv still pending

**Review by NikosTsiligaridis — APPROVED** [review]

**Pull request merged**